plasmicpkgs/antd/src/registerRate.ts

```diff
@@ -0,0 +1,65 @@
+import registerComponent, {
+  ComponentMeta,
+} from "@plasmicapp/host/registerComponent";
+import { Rate, RateProps } from "antd";
+import { Registerable } from "./registerable";
+
+
+export const rateMeta: ComponentMeta<RateProps> = { 
+name: "AntdRate",
+displayName: "Antd Rate",
+props: {
+    allowClear: {
+        type: "boolean",
+        description: "Whether to allow clear when click again",
+        defaultValue: true
+    },
+    allowHalf: {
+        type: "boolean",
+        description: "Whether to allow semi selection",
+        default: false
+    },
+    autoFocus: {
+        type: "boolean",
+        description: "If get focus when component mounted",
+        default: false
+    },
+    count: {
+        type: "number",
+        description: "Star count"
+    },
+    className: {
+        type: "string",
+        description: "The custom class name of rate"
+    },
+    defaultValue: {
+        type: "number",
+        description: "The default value",
+        default: 0
```

Comment:
Instead of having both defaultValue and value, we usually have only a single `value` prop with this configuration:
```
    value: {
      type: "number",
      description: "The default value",
      editOnly: true,
      uncontrolledProp: "defaultValue",
      defaultValueHint: 0
    },